.env

```diff
@@ -3,6 +3,7 @@ FLASK_DEBUG=true
 DEVEL=true
 SECRET_KEY=insecure_dev_key
 CONTRACTS_API_URL=https://contracts.staging.canonical.com/
+CANONICAL_CLA_API_URL=https://cla.staging.canonical.com/
```

Comment:
Do you mean staging?

Reply:
Yes, when running this project locally it should point the staging environment by default.
Not sure if that answers the question.